src/docs/BUILD.md

```diff
@@ -37,6 +37,30 @@ The preferred approach is to build both OpenVINO and OpenVINO GenAI from sources
 
 ## Build Instructions
 
+### Build OpenVINO GenAI as OpenVINO Extra Module
+
+OpenVINO GenAI can be built as an extra module during the OpenVINO build process. This method simplifies the build process by integrating OpenVINO GenAI directly into the OpenVINO build.
+
+1. Clone OpenVINO repository:
+    ```sh
+    git clone --recursive https://github.com/openvinotoolkit/openvino.git
+    ```
+2. Clone OpenVINO GenAI repository:
+    ```sh
+    git clone --recursive https://github.com/openvinotoolkit/openvino_genai.git
```

Comment:
maybe we can merge 2 git clones steps into a single one?
E.g. clone repos with 2 sub-commands

Reply:
Fixed